scripts/setup-firebase-flavors.sh

```
@@ -0,0 +1,185 @@
+check_flutterfire_cli() {
+    if ! command -v flutterfire &> /dev/null; then
+        print_error "flutterfire CLI is not installed. Please install it first:"
+        echo "dart pub global activate flutterfire_cli"
+        exit 1
+    fi
+}
+
+# Function to ensure we're in the app directory
+ensure_app_directory() {
+    # Get the script's directory
+    local script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)